.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
@ulucinar during our discussion, I said that we don't need to cherry-pick github dir to release branch but we might need it to have CI running on that branch. So, I cherry-picked it as well, since there is no reason not to do so :)